.tool-versions

```diff
@@ -0,0 +1 @@
+php 7.4.16
```

Comment:
What is the purpose of this file?

Reply:
This files installs the correct version of PHP using [asdf](https://asdf-vm.com/#/). Someone new coming to the project can simply `cd monica && asdf install` and all of the correct runtime versions (node + php) will be setup for them.